nntrainer/tensor/cl_operations/blas_kernel_interface.cpp

```diff
@@ -200,38 +200,47 @@ void multiplyCl(Tensor &input, float const &value) {
   }
 }
 
-void add_i_cl(Tensor const &input, Tensor &result) {
+void add_i_cl(Tensor &inputA, Tensor const &inputB) {
 
-  CREATE_IF_EMPTY_DIMS(result, result.getDim());
+  CREATE_IF_EMPTY_DIMS(inputA, inputA.getDim());
```

Comment:
I have a question. The result tensor before modification could be empty, but is it possible that inputA tensor is empty in the current modification?

Reply:
Right! It shouldn't be. I'll modify it, thanks.